src/Uno.UWP/System.IO/WindowsRuntimeStreamExtensions.cs

```diff
@@ -9,10 +10,30 @@ public static class WindowsRuntimeStreamExtensions
 		public static IInputStream AsInputStream(this Stream stream) { throw new NotImplementedException(); }
 		[Uno.NotImplemented]
 		public static IOutputStream AsOutputStream(this Stream stream) { throw new NotImplementedException(); }
-		[Uno.NotImplemented]
-		public static IRandomAccessStream AsRandomAccessStream(this Stream stream) { throw new NotImplementedException(); }
-		[Uno.NotImplemented]
-		public static Stream AsStream(this IRandomAccessStream windowsRuntimeStream) { throw new NotImplementedException(); }
+
+		public static IRandomAccessStream AsRandomAccessStream(this Stream stream)
+		{
+			if (stream is null)
+			{
+				throw new ArgumentNullException(nameof(stream));
+			}
+			return new RandomAccessStreamWrapper(stream);
+		}
+
+		public static Stream AsStream(this IRandomAccessStream stream)
+		{
+			if (stream is null)
+			{
+				throw new ArgumentNullException(nameof(stream));
+			}
+
+			if (stream is RandomAccessStreamWrapper wrapper)
+			{
+				return wrapper.GetSourceStream();
+			}
+			else throw new ArgumentException($"stream must be a {nameof(RandomAccessStreamWrapper)}", nameof(stream));
```

Comment:
Braces even for single line conditions

Reply:
Documented over at https://github.com/unoplatform/uno/pull/1212/files#diff-336e4bcde45e4393a322cd34f381311dR35